source/_posts/2017-06-30-uefi-slash-edk-ii-corebootpkg-on-pc-engines-apu2.markdown

```diff
@@ -14,15 +14,15 @@ supporting GPT. At that point I was not sure if I can utilize GRUB GPT support
 on APU2, but this led to other questions:
 
 - Is it possible to boot UEFI-aware OS on PC Engines APUx boards ?
-- What level of security I can get with UEFI-aware OS in comparison to coreboot ?
+- What level of security can I get with UEFI-awere OS in comparison to coreboot ?
```

Comment:
@miczyg1 I almost missed that comment. Please always mention me. Should be fixed now.